toolbox/reminder_scheduler/reminder_lsoa.py

```diff
@@ -41,11 +41,24 @@ def main(lsoa_file_path: Path, reminder_action_type: str, action_plan_id: uuid.U
 
 
 def build_action_rule_classifiers(lsoas):
+    if not all(check_lsoa(row, lsoa) for row, lsoa in enumerate(lsoas, 1)):
+        print('INVALID FILE, EXITING')
+        exit(1)
     lsoa_classifier_clause = "', '".join(lsoas)
     return f"case_type = 'HH' " \
            f"AND lsoa IN ('{lsoa_classifier_clause}')"
 
 
+def check_lsoa(row, lsoa):
+    if not lsoa.isalnum():
+        print(f'Row: {row}, LSOA {repr(lsoa)} is not alphanumeric')
+        return False
+    if len(lsoa) > 9:
```

Comment:
Do we know the exact length an LSOA should be? I'm not concerned about this bit of validation since it won't break the SQL but if we know an exact length it should be we could check the length is equal.

Reply:
Hmm good point about the unexpected length. I took the length straight from the [sample validator](https://github.com/ONSdigital/census-rm-sample-loader/blob/ce0c86ac180dbdf46ab2744a4ef7874efec00171/validate_sample.py#L51). Will check with Jenny to see if there's an exact length. If not, I'll remove this bit.

Reply:
Jenny confirmed we're waiting on confirmation on exact format of this field (and similar geographic fields). I'll remove this length check and pop a ticket on the board to revisit

Reply:
Confirmed LSOA length will always be 9 chars. Will update the check 👍

Reply:
As discussed, we will keep the LSOA length check to match what is set in the [sample validator](https://github.com/ONSdigital/census-rm-sample-loader/blob/ce0c86ac180dbdf46ab2744a4ef7874efec00171/validate_sample.py#L51.